[Term Entry] Kotlin Math .exp()

vaisrinivasan opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

I created the exp.md in Kotlin > concepts > math-methods > terms directory.
Closes #3523 

### Type of Change

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

vaisrinivasan commented:
Hi. Need some help with the markdown part (The initial one). 
`---
Title: 'sqrt()'
Description: 'Calculates the square root of a given number.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Math'
  - 'Functions'
    CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---`
Whichever solution I try, it doesn't come as a table like given in the template and hence the test is failing.

caupolicandiaz commented on content/kotlin/concepts/math-methods/terms/exp/exp.md:
```diff
@@ -0,0 +1,44 @@
+---
+Title: 'exp()'
+Description: 'Calculates the Euler's number e raised to the power of the value of a given number.'
```
```suggestion
Description: 'Returns e raised to the power of a given number.'
```

caupolicandiaz commented on content/kotlin/concepts/math-methods/terms/exp/exp.md:
```diff
@@ -0,0 +1,44 @@
+  - 'paths/computer-science'
+---
+
+The **`exp()`** method in Kotlin's `math` class is used to calculate the Euler's number e raised to the power of the value of the given number.
```
```suggestion
The **`exp()`** method in Kotlin's `math` class is used to calculate Euler's number *e* raised to the power of a given number.
```

caupolicandiaz commented on content/kotlin/concepts/math-methods/terms/exp/exp.md:
````diff
@@ -0,0 +1,44 @@
+exp(number)
+```
+
+- `number`: The number for which you want to calculate the Euler's number e raised to the power. It should be of type `Double` or `Float`.
````
```suggestion
- `number`: A number of type `Double` or `Float`. The method returns a number, of the same type given.
```

caupolicandiaz commented on content/kotlin/concepts/math-methods/terms/exp/exp.md:
```diff
@@ -0,0 +1,44 @@
+
+- `number`: The number for which you want to calculate the Euler's number e raised to the power. It should be of type `Double` or `Float`.
+
+The method returns a number, of the same type given.
```
```suggestion
```

caupolicandiaz commented on content/kotlin/concepts/math-methods/terms/exp/exp.md:
```diff
@@ -0,0 +1,44 @@
+
+## Example
+
+This example shows how to use the `exp()` method to calculate the Euler's number e raised to the power of the value of the given number.
```
```suggestion
This example demonstrates a basic implementation of the `exp()` method.
```

caupolicandiaz commented on content/kotlin/concepts/math-methods/terms/exp/exp.md:
```diff
@@ -0,0 +1,44 @@
+fun main() {
+    val number = 3.0
+    val ePowNumber = exp(number)
+    println("The Euler's number e raised to the power of $number is $ePowNumber.")
```
```suggestion
    println("Euler's number e raised to the power of $number is $ePowNumber.")
```

caupolicandiaz commented on content/kotlin/concepts/math-methods/terms/exp/exp.md:
````diff
@@ -0,0 +1,44 @@
+The code above will result in the following output:
+
+```shell
+The Euler's number e raised to the power of 3.0 is 20.085536923187668.
````
```suggestion
Euler's number e raised to the power of 3.0 is 20.085536923187668.
```

caupolicandiaz reviewed (changes requested):
@vaisrinivasan I've completed my initial review. Please update the entry, and tag me when you've made the changes. Thanks

vaisrinivasan commented:
@caupolicandiaz : I've addressed the review comments and all the checks are passing. Please re-review.
I haven't done the below checks as the review is already happening in this PR. Let me know if I've to push from a branch named after the changes I'm making and I'll be happy to accommodate the same.

 I have confirmed my changes are not being pushed from my forked main branch.
 I have confirmed that I'm pushing from a new branch named after the changes I'm making.

caupolicandiaz reviewed (approved):
@vaisrinivasan This is ready for a second review.

vaisrinivasan commented:
Hi. Can I get a second review for this? Thanks.

caupolicandiaz commented:
> Hi. Can I get a second review for this? Thanks.

Please be patient, there are many PRs and few maintainers

SSwiniarski reviewed (approved):
@vaisrinivasan Looks good.
